<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplingScheduleUI</class>
 <widget class="QWidget" name="SamplingScheduleUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>待处理任务单</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="refleshBtn">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyTableView" name="taskToScheduledView"/>
      </item>
      <item>
       <widget class="SqlPageControleUI" name="pageCtrl" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>已安排任务单</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="myScheduleBtn">
          <property name="text">
           <string>我的排单</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="myTaskBtn">
          <property name="text">
           <string>我的采样</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="allBtn">
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>人员任务查询：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MyTableView" name="taskScheduledView"/>
      </item>
      <item>
       <widget class="SqlPageControleUI" name="pageCtrl2" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>mytableview.h</header>
  </customwidget>
  <customwidget>
   <class>SqlPageControleUI</class>
   <extends>QWidget</extends>
   <header>../sqlPageControleUI/sqlpagecontroleui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
